Bottom: b63072d91a0b2fdcd2adce6c4fca24952dcd4abd
Top:    4b22788c894da457c3da8de419e4e14d00bad19c
Author: scott snyder <sss@karma>
Date:   2020-02-01 10:37:02 -0500

gitignore additions


---

diff --git a/.gitignore b/.gitignore
index d9d3967a12c..dd80578383e 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,4 @@
+NOTES
 *.diff
 *.patch
 *.orig
